pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerBuilderImpl.java

```diff
@@ -40,9 +44,9 @@
     private static final long serialVersionUID = 1L;
 
     private final PulsarClientImpl client;
-    private String topicName;
     private String subscriptionName;
     private final ConsumerConfiguration conf;
+    private List<String> topicNames;
```

Comment:
I would suggest using Set instead of List, so this allows us removing duplicated topic names.